frontend/src/components/viewers/Tensorboard.tsx

```diff
@@ -64,6 +64,9 @@ interface TensorboardViewerState {
   tensorflowVersion: string;
 }
 
+// TODO(jingzhang36): we'll later parse Tensorboard version from mlpipeline-ui-metadata.json file.
+const DefaultTensorboardVersion = '2.0.0';
```

Comment:
Convention is to name constants as `const DEFAULT_TENSORBOARD_VERSION = '2.0.0';`

Reply:
done